### PR TITLE
Update do.m3u

### PR DESCRIPTION
**Pull request by Pecaquito**

### CODE REVIEW

File: streams/do.m3u [hunk 1]
```diff
@@ -146,7 +146,7 @@ https://ss3.domint.net:3136/gtv_str/globalhd/playlist.m3u8
 #EXTINF:-1 tvg-id="GuacaraTV.do",Guacara TV (720p)
 https://imagenuniversaltv.net:3820/live/guacaratvlive.m3u8
 #EXTINF:-1 tvg-id="GuaymateTV.do",Guaymate TV (360p)
-https://5790d294af2dc.streamlock.net/8100/8100/playlist.m3u8
```
Comment (PopeyeTheSai10r) [inline streams/do.m3u:149]: This stream still works. 
Can't tell whether the programing is for Guaymate TV or not.

File: streams/do.m3u [hunk 2]
```diff
@@ -395,12 +399,18 @@ https://ss9.domint.net:3042/gstv_str/globalsocialtv/playlist.m3u8
 https://cloud5.idealtv.net/hls/live.m3u8
 #EXTINF:-1 tvg-id="",La Hora TV
 https://ss5.domint.net:3056/lhtv_str/lahoratv/playlist.m3u8
+#EXTINF:-1 tvg-id="",La Ruta TV
+https://vsrv2.az-streamingserver.com:3983/live/larutatvlive.m3u8
```
Comment (PopeyeTheSai10r) [inline streams/do.m3u:403]: Geo-locked or Offline.

File: streams/do.m3u (hunk 1 above)
Comment (Pecaquito) [reply streams/do.m3u:149]: Yes , the link had changed, there is different channels on that Link

Comment (Dum4G) [reply streams/do.m3u:149]: Whatever that was it's currently shows 404. Not 24/7

Comment (Dum4G) [reply streams/do.m3u:149]: image

Comment (Pecaquito) [reply streams/do.m3u:149]: Many channel front Dominican Republic when off line last week and some then changed they link, you can tell by the logo, that the original channel is on the other link, now the channel that come back on that link after off time was different channel with different logo, I had see this before on this streamlock link

Comment (Pecaquito) [reply streams/do.m3u:149]: I forgot to mention, that link used to be Farovision tv , like 4 years ago, then they changed to Guaymate, i juts look at it on a old playlist that I had saved from 2019.

**Comment by GUAYMATE71** [comment]
https://glb.bozztv.com/glb/ssh101/guaymatetv/index.m3u8
EL CAFE DE LA TERDE (1)

**Comment by GUAYMATE71** [comment]
https://glb.bozztv.com/glb/ssh101/guaymatetv/index.m3u8

**Pull request closed without merging**